storage/src/persistent/database.rs

```diff
@@ -123,6 +123,22 @@ pub trait KeyValueStoreWithSchema<S: KeyValueSchema> {
     fn get_mem_use_stats(&self) -> Result<RocksDBStats, DBError>;
```

Comment:
@mat13mn btw with the new trait for this method, it should be removed from here (and the duplicated implementation)